src/components/Text/Text.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {render} from '@testing-library/react';
+import Text from './Text';
+
+const TEXT = 'some text';
+
+describe('Text element', () => {
+  it('Applies "Text--large" className if props.fontSize is "large"', () => {
+    const {container} = render(<Text fontSize="large">{TEXT}</Text>);
+    expect(container.firstChild.classList.contains('Text--large')).toBeTruthy();
```

Comment:
Can we use `toBeTrue` for these conditions instead of `toBeTruthy`? `contains` should return either `true` or `false`, so we should be setting a hard expectation that it should return true if the class list contains a class (or doesn't.)

Reply:
I updated tests. Please let me know if you have any additional thoughts on that